### PR TITLE
Adding code to get licensed applications only, as opposed to all apps

### PR DESCRIPTION
**Pull request by sergeib-gmail**
Regions specific changes for only getting licensed application data

### CODE REVIEW

File: src/main/java/com/contrastsecurity/http/UrlBuilder.java [hunk 1]
```diff
@@ -39,6 +39,10 @@ public String getApplicationsUrl(String organizationId) {
         return String.format("/ng/%s/applications?%s", organizationId, "base=false");
     }
 
+    public String getLicensedApplicationsUrl(String organizationId) {
+        return String.format("/ng/%s/applications%s", organizationId, "/filter?expand=license,compliance_policy,skip_links,technologies,trace_severity_breakdown,metadata&sort=appName&quickFilter=LICENSED");
```
Comment (gmoore) [inline src/main/java/com/contrastsecurity/http/UrlBuilder.java]: Can we cut out some(or most) of these expand parameters. Expands really hurt performance.

Comment (sergeib-gmail) [reply src/main/java/com/contrastsecurity/http/UrlBuilder.java]: Yep, took out all the expands, as the script does not seem to need that data. Will push changes shortly.

**Comment by gmoore** [comment]
Hey @sergeib-gmail when I run the following code I get an NPE. How are you using this new method? 

```
            apps = contrastSDK.getLicensedApplications(orgUuid);
            for (Application app : apps.getApplications()) {
                System.out.println(app.getName());
                System.out.println(app.getLicense().getLevel());
            }
```

**Comment by gmoore** [comment]
Oh wait I get it, when you removed the expand param, you removed all of them. Add back only the `license` expand and it should work.

Also feel free to lmk if I'm just missing something.

**Comment by sergeib-gmail** [comment]
Hi @gmoore - I just need a list of all licensed apps,  after that I have this code in my reporting script to get those 5 attributes:

app.getName(), app.getId(), app.getStatus(), app.getArchived(), app.getTags()

I don't need to check for the app.getLicense(), the endpoint is already returning only licensed applications because of the quickFilter=LICENSED in this method in URLBuilder class.

 public String getLicensedApplicationsUrl(String organizationId) {
        return String.format("/ng/%s/applications%s", organizationId, "/filter?sort=appName&quickFilter=LICENSED");
    }

I am happy to put the expand back in though, if you think it's better with it.

**Comment by gmoore** [comment]
Oh interesting, that makes sense. All the same, put the parameter back in. Feels strange to get licensed apps and not the licenses 🚀

**Pull request merged**